waku/waku_api/rest/admin/handlers.nim

```diff
@@ -0,0 +1,111 @@
+when (NimMajor, NimMinor) < (1, 4):
+  {.push raises: [Defect].}
+else:
+  {.push raises: [].}
+
+import
+  std/strformat,
+  std/sequtils,
+  stew/byteutils,
+  chronicles,
+  json_serialization,
+  presto/route,
+  libp2p/[peerinfo, switch]
+
+import
+  ../../../waku_core,
+  ../../../waku_store,
+  ../../../waku_filter,
+  ../../../waku_relay,
+  ../../../waku_node,
+  ../../../node/peer_manager,
+  ../responses,
+  ../serdes,
+  ./types
+
+export types
+
+logScope:
+  topics = "waku node rest admin api"
+
+const ROUTE_ADMIN_V1_PEERS* = "/admin/v1/peers"
+
+func decodeRequestBody[T](contentBody: Option[ContentBody]) : Result[T, RestApiResponse] =
+  if contentBody.isNone():
+    return err(RestApiResponse.badRequest("Missing content body"))
+
+  let reqBodyContentType = MediaType.init($contentBody.get().contentType)
+  if reqBodyContentType != MIMETYPE_JSON:
+    return err(RestApiResponse.badRequest("Wrong Content-Type, expected application/json"))
+
+  let reqBodyData = contentBody.get().data
+
+  let requestResult = decodeFromJsonBytes(T, reqBodyData)
+  if requestResult.isErr():
+    return err(RestApiResponse.badRequest("Invalid content body, could not decode. " &
+                                          $requestResult.error))
+
+  return ok(requestResult.get())
+
+proc installAdminV1GetPeersHandler(router: var RestRouter, node: WakuNode) =
```

Comment:
This `proc` is very interesting. I wonder if we better return more information in the response so that it is clear which protocols each peer supports.

Something like:
```
{
  "peers_info": [
    {
      "protocol": "store",
      "peers": ["", "", "etc"]
    },
    {
      "protocol": "admin",
      "peers": ["", "..."]
    },
    {
      "protocol": "lightpush",
      "peers": ["", "..."]
    }
  ]
}
```

Reply:
I would do maybe the other way around, group protocols by peers. 
Shall we discuss in a new issue taken?

Reply:
> I would do maybe the other way around, group protocols by peers. Shall we discuss in a new issue taken?

It sounds great to perform "group protocols by peers".
If we don't apply that within this PR, we need to create an issue so that we don't miss that. It's up to you, what you consider better :)
Thanks!

Reply:
Ok, its there! @Ivansete-status please check!